jobs/config.json

```diff
@@ -187,6 +187,13 @@
   ]
 },
 
+"ci-kubernetes-node-memcg-serial": {
+  "scenario": "kubernetes_kubelet",
+  "args": [
+    "--properties=test/e2e_node/jenkins/jenkins-memcg-serial.properties"
+  ]
+},
+
```

Comment:
ack, that's what I thought as well :-) just unsure since there are two different job names around, reviewed based off node jobs.